scripts/seedDb.mjs

```diff
@@ -2,6 +2,7 @@ import amazingRace35Data from "../app/leagueData/AmazingRace_35.js"
 import amazingRace36Data from "../app/leagueData/AmazingRace_36.js"
 import amazingRace37Data from "../app/leagueData/AmazingRace_37.js"
 import bigBrother26Data from "../app/leagueData/BigBrother_26.js"
+import survivor47data from "../app/leagueData/Survivor_47.js"
```

Comment:
This is lower case "data", compared to the rest of the capitalized "Data"

Reply:
You mean like this...
```suggestion
import survivor47Data from "../app/leagueData/Survivor_47.js"
```

Reply:
Yep!